lib/util.js

```diff
@@ -410,7 +394,7 @@ function formatValue(ctx, value, recurseTimes) {
     });
   }
 
-  var constructor = getConstructorOf(value);
+  var constructor = internalUtil.getConstructorOf(value);
```

Comment:
We have done just that in 9077b482718b8bc6cafcabcd6b84e99498918b26 / https://github.com/nodejs/node/pull/11404.